### ISSUE TITLE
Add an option for the Twist and Infinitum not to open the acknowledgement on right click

### ISSUE THREAD

**Issue by nddragoon** [Extegral/Enigmatic-Legacy#430]
**Provide a detailed and clear description of feature or change that you want to offer.**
a config option or maybe an ingame toggle to make the twist and infinitum not show the acknowledgement when right clicking with them

**Describe why you want this. If there's a reason.**
since the twist and infinitum's main reason to exist is as weapons, constantly opening the acknowledgement when using them as such is very annoying, especially with things from other mods that can add right click functionality to the infinitum, like the ender enchantment from cyclic, the teleportation enchantment from travel anchors, or certain apotheosis affixes

**Comment by Aizistral** [comment]
Makes sense. Although rather than adding a config option, I think it'd be better if I added some way to switch this behavior for individual books in-game, say, putting them through a crafting grid with no other items.

**Comment by Toastr125** [comment]
Doesn't that functionality already get disabled when holding something in your off-hand?

**Comment by nddragoon** [comment]
> Doesn't that functionality already get disabled when holding something in your off-hand?

yeah but many people myself included don't like using shields and stuff, it can often mess with other stuff you're trying to do
